tests/integration/qualification/loadbalancing_test.go

```diff
@@ -61,12 +62,22 @@ func TestIngressLoadBalancing(t *testing.T) {
 
 	g := galley.NewOrFail(t, ctx, galley.Config{})
 
-	d := bookinfo.DeployOrFail(t, ctx, bookinfo.Config{Cfg: bookinfo.BookInfo})
-	g.ApplyConfigOrFail(t,
+	bookinfoNs, err := namespace.New(ctx, "istio-bookinfo", true)
+	if err != nil {
+		t.Fatalf("Could not create istio-bookinfo Namespace; err:%v", err)
+	}
+	d := bookinfo.DeployOrFail(t, ctx, bookinfo.Config{Namespace: bookinfoNs, Cfg: bookinfo.BookInfo})
+
+	g.ApplyConfigOrFail(
+		t,
+		d.Namespace(),
+		bookinfo.NetworkingBookinfoGateway.LoadOrFailGatewayFileWithNamespace(t, bookinfoNs.Name()))
+	g.ApplyConfigOrFail(
+		t,
 		d.Namespace(),
-		bookinfo.NetworkingBookinfoGateway.LoadOrFail(t),
-		bookinfo.NetworkingDestinationRuleAll.LoadOrFail(t),
-		bookinfo.NetworkingVirtualServiceAllV1.LoadOrFail(t))
+		bookinfo.GetDestinationRuleConfigFile(t, ctx).LoadOrFailWithNamespace(t, bookinfoNs.Name()),
```

Comment:
nit: to be consistent with other method names, this should probably be `LoadWithNamespaceOrFail`

Reply:
Fixing in PR: https://github.com/istio/istio/pull/11209/